playfab-docs/features/data/dataconnections/data-connection-adx.md

```diff
@@ -55,16 +55,14 @@ Once your Azure AD app is listed as a “Database Admin”, you can move on to c
 | Cluster URI  | The ADX cluster URI.|
 | Database | The database name. The PlayFab ingestion service will create a table in this database and ingest the data to that table. |
 | Table | The name of the table in the database that PlayFab will ingest the data into. If the table does not exist, then the ingestion service will create the table. |
-| Application ID | The Azure AD App ID. |
-| Tenant ID | The Azure AD Tenant ID. |
 
-![Screenshot of configuring and ADX data connection](media/configure-new-data-connection.png "ADX data connection configuration") 
+![Screenshot of configuring and ADX data connection](media/Config-ADX-Data-Connections.png "ADX data connection configuration") 
 
 > [!Note]
 > We recommend using a table name that does not exist. If you use an existing table and the schema is not what PlayFab is expecting, then the schema will be modified. 
 
-### Step 3: Create and Share a Client Secret 
```

Comment:
General comments for the page:

To make it easier to follow.

1) Create a new ADX resource in Azure
2) Link the newly created ADX to PlayFab using Game Manager
3) Show me how to use the connection to get something going :)

Reply:
Use "select" instead of "click"

Reply:
Regarding the commands - There is a 'copy` command in the setup window, so if I spell out the commands in the set up instructions, it would be more cumbersome to follow along - especially because these commands are uniquely generated based on the information entered in the previous fields. The commands include the 'keys' and database name etc. so it's less work just to copy paste them. I've updated docs to inform the reader of the availability of 'copy' icon.